pavelib/i18n.py

```diff
@@ -74,11 +71,13 @@ def i18n_dummy():
     Simulate international translation by generating dummy strings
     corresponding to source strings.
     """
-    cmd = "i18n_tool dummy"
-    sh(cmd)
+    sh("i18n_tool dummy")
     # Need to then compile the new dummy strings
-    cmd = "i18n_tool generate"
-    sh(cmd)
+    sh("i18n_tool generate")
+
+    # Generate static i18n JS files.
+    for system in ['lms', 'cms']:
+        sh(django_cmd(system, DEFAULT_SETTINGS, 'compilejsi18n'))
```

Comment:
I've created a pull request to address this: https://github.com/edx/edx-platform/pull/13211